src/it/projects/MJAVADOC-498_mm_modulepath/invoker.properties

```diff
@@ -16,3 +16,4 @@
 # under the License.
 
 invoker.java.version = 9+
+invoker.goals=clean compile javadoc:javadoc
```

Comment:
clean is not necessary - invoker always clear project directory - https://maven.apache.org/plugins/maven-invoker-plugin/integration-test-mojo.html#cloneClean

Reply:
Awesome, didn't know. This is done in many ITs actually. Waste of cycles. Will change.